tst/CTA.WebForms2Blazor.Tests/FileConverterTests.cs

```diff
@@ -111,5 +191,61 @@ public async Task TestWebConfigFileConverter()
             Assert.IsTrue(fi.RelativePath.Equals(relativePath));
 
         }
+        
+        private CodeAnalyzer CreateDefaultCodeAnalyzer()
+        {
+            // Create a basic logger
+            var loggerFactory = LoggerFactory.Create(builder => builder.SetMinimumLevel(LogLevel.Debug).AddConsole());
+            var logger = loggerFactory.CreateLogger("");
+
+            // Set up analyzer config
+            var configuration = new AnalyzerConfiguration(LanguageOptions.CSharp)
+            {
+                ExportSettings = {GenerateJsonOutput = false},
+                MetaDataSettings =
+                {
+                    ReferenceData = true,
+                    LoadBuildData = true,
+                    GenerateBinFiles = true,
+                    LocationData = false,
+                    MethodInvocations = false,
+                    LiteralExpressions = false,
+                    LambdaMethods = false,
+                    DeclarationNodes = false,
+                    Annotations = false,
+                    InterfaceDeclarations = false,
+                    EnumDeclarations = false,
+                    StructDeclarations = false,
+                    ReturnStatements = false,
+                    InvocationArguments = false,
+                    ElementAccess = false,
+                    MemberAccess = false
+                }
+            };
+
+            return CodeAnalyzerFactory.GetAnalyzer(configuration, logger);
+        }
+
+        [Test]
+        public async Task TestProjectFileConverter()
+        {
+            FileConverter fc = new ProjectFileConverter(WebForms2BlazorTestFixture.EShopOnBlazorSolutionPath,
+                WebForms2BlazorTestFixture.EShopLegacyWebFormsProjectPath,
+                _blazorWorkspaceManager, _webFormsProjectAnalyzer);
+
+            IEnumerable<FileInformation> fileList = await fc.MigrateFileAsync();
+            FileInformation fi = fileList.First();
```

Comment:
If you're only expecting one file, you might want to use `Single()` instead of `First()` that way it will throw an error if more than one file information object is present, but not entirely necessary to make this change